tools/preflight/preflight.sh

```diff
@@ -4,6 +4,8 @@
 
 set -o pipefail
 
+TIMESTAMP=$(date +%F:%M:%S)
```

Comment:
instead of `:` use `-`. Use this one `date +%F_%H-%M-%S`

Reply:
resolved